db/seeds.rb

```diff
@@ -1,7 +1,15 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+User.create!(name: "Example User",
+            email: "example@railstutorial.org",
+            password: "foobar",
+            password_confirmation: "foobar",
+            admin: true)
+
+50.times do |n|
+  name = n
+  email = "example-#{n+1}@railstutorial.org"
+  password = "password"
+  User.create!(name: name,
```

Comment:
create vs create! khác j nhau e

Reply:
create tương đương với .new theo sau là .save.
create! tương đương với .new theo sau là .save! (in lỗi nếu lưu thất bại)